osbs/build/build_request.py

```diff
@@ -1676,6 +1691,7 @@ def render(self, validate=True):
         self.render_version()
         self.render_node_selectors()
         self.set_reactor_config_map()
+        self.set_reactor_config_yaml()
```

Comment:
This method should be combined with `set_reactor_config_map` to avoid setting `REACTOR_CONFIG` twice.

```python
if self.spec.reactor_config_yaml.value:
  # set REACTOR_CONFIG with value
elif self.spec.reactor_config_map.value:
  # set REACTOR_CONFIG with valueFrom
```

I don't think we necessarily need to restrict `reactor_config_yaml` to just worker. If it's there use it. `reactor_config_yaml` takes precedence over `reactor_config_map`.

Reply:
ok